src/code_802A3730.c

```diff
@@ -150,18 +150,18 @@ void func_802A38B4(void) {
 
     switch(D_80150124) {
         case 0:
```

Comment:
@MegaMech I think you had the right idea about quit out values for the 'duplicated' menu options. I haven't thoroughly tested but D_80150124 seems to be getting set to the duplicate value when quitting to course change or driver change from game.

Reply:
Now that we know that. I think we should put that in the name. Something along the lines of:

START_MENU_FROM_QUIT

Reply:
And it gets set to what I called UNUSED_MENU (4) when retrying from time trials... think I'll have to investigate some more...